src/DataGridBundle/Tests/Export/GridRowExtractorTest.php

```
@@ -0,0 +1,100 @@
+
+        self::assertSame(['id', 'name'], array_keys($row));
+        self::assertSame($ulid->toBase58(), $row['id']);
+        self::assertSame('Acme', $row['name']);
+    }
+
+    public function testEntityIdIsPresentEvenWhenGridHasNoIdColumn(): void
+    {
+        $ulid = new Ulid();
+        $entity = new class($ulid) {
+            public function __construct(
+                public Ulid $id
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)